starter-code/contact-app/src/app/contact-list/contact-list.component.html

```diff
@@ -1,18 +1,34 @@
-<div>
-  <label for="contact-name"></label>
+<div class="input-container">
+  <label for="contact-name">NAME</label>
   <input type="text" [(ngModel)]="newContact.name">
 </div>
-<div>
-  <label for="contact-email"></label>
+<div class="input-container">
+  <label for="contact-email">EMAIL</label>
   <input type="text" [(ngModel)]="newContact.email">
 </div>
-<div>
-  <label for="contact-phone"></label>
+<div class="input-container">
+  <label for="contact-phone">PHONE</label>
   <input type="text" [(ngModel)]="newContact.phoneNumber">
 </div>
-<div>
-  <label for="contact-image"></label>
+<div class="input-container">
+  <label for="contact-image">IMAGE</label>
   <input type="text" [(ngModel)]="newContact.image">
 </div>
 
-<button (click)="addContact">Submit New Contact</button>
+<button (click)="addContact()">Submit New Contact</button>
+
+<ul>
+  <li *ngFor="let contact of contacts">
+    <div class="image-container">
+      <img [src]="contact.image" alt="image">
```

Comment:
It is best practice to always include descriptive alt text (if the image is not purely decorative), e.g.: ```alt="Profile picture of {{ contact.name }}"```